django_dramatiq/models.py

```diff
@@ -10,9 +12,19 @@
 #: The database label to use when storing task metadata.
 DATABASE_LABEL = DjangoDramatiqConfig.tasks_database()
 
+DJANGO_DRAMATIQ_TASKS_NOT_WRITES = getattr(settings, "DJANGO_DRAMATIQ_TASKS_NOT_WRITES", [])
+DJANGO_DRAMATIQ_TASKS_WRITES_ONLY = getattr(settings, "DJANGO_DRAMATIQ_TASKS_WRITES_ONLY", [])
+
 
 class TaskManager(models.Manager):
-    def create_or_update_from_message(self, message, **extra_fields):
+    def create_or_update_from_message(self, message, **extra_fields) -> Optional['Task']:
```

Comment:
**nitpick:** 
```suggestion
    def create_or_update_from_message(self, message, **extra_fields):
```

I dont believe we are using types in this project. Can you please remove this for consistency? 😄

Reply:
I suggest adding types everywhere, it always makes it easier to read the code. Compatibility will not be affected: in the library "Requirements Python 3.9+"